extmod/btstack/modbluetooth_btstack.c

```diff
@@ -1042,6 +1048,24 @@ int mp_bluetooth_gatts_set_buffer(uint16_t value_handle, size_t len, bool append
     return mp_bluetooth_gatts_db_resize(MP_STATE_PORT(bluetooth_btstack_root_pointers)->gatts_db, value_handle, len, append);
 }
 
+int mp_bluetooth_get_preferred_mtu(void) {
+    if (!mp_bluetooth_is_active()) {
+        return ERRNO_BLUETOOTH_NOT_ACTIVE;
```

Comment:
this should probably return a negative number on error (see below)

Reply:
We already have other places where modbluetooth_{btstack,nimble}.c raises rather than returning, so I think that makes sense here.

(I'd like to go through and standardise this everywhere)

Reply:
Ok.  This is low priority so can be fixed later if that's easier.